### ISSUE TITLE
[#SomeChanges Suggestion] Judgements of the Wise should gain mana when judgement is fully absorbed

### ISSUE THREAD

**Issue by Provim** [ClassicWoWCommunity/cata-classic-bugs#747]
### Description
Judgements of the Wise should gain mana when judgement is fully absorbed. 

It was part of 3.3.5b Arena-Tournament patch and it made ret paladins life easier vs priests, shamans, mages and even big sacred shield holy paladins that absorbed whole judgement.

Pros: 
- Won't affect PvE, the only fight this change would affect I can think of is Twin Valkyrs during shield special ability,

Cons:
- I do not see any.

**Comment by Evolvee** [comment]
yes, Preg is too weak, needs a [buff](https://i.imgur.com/0xikrJN.png)

**Comment by Azmoddan** [comment]
Removes outplay potential. Unwelcome and unwarranted change.

**Comment by Dverikk** [comment]
Bad change, preg already at the top. Why do you even want to buff them more?

**Comment by JamminL** [comment]
Archiving old non-relevant issues in preparation for MOP. If I made a mistake please comment on it.
